addons/web/static/src/less/fields_extra.less

```diff
@@ -6,7 +6,8 @@
 // Required
 .o_required_modifier {
     &.o_input, .o_input {
-        background-color: #D2D2FF;
+        background-color: #D2D2FF !important; //This color does not have to be
+                                              //overwritten by grid or over view.
```

Comment:
Just implicit conventions:
- No space before `!important`
- Space after `//`

Also I am not sure to understand the "over view" part in the comment. Maybe the whole comment is not important...